featurebyte/models/feature.py

```diff
@@ -226,7 +230,7 @@ def _add_derived_attributes(cls, values: dict[str, Any]) -> dict[str, Any]:
         if any(not x for x in derived_attributes):
             # only derive attributes if any of them is missing
             # extract table ids & entity ids from the graph
-            graph = values["graph"]
+            graph = QueryGraph(**values["internal_graph"])
```

Comment:
When using `root_validator` (`pre=False`), pydantic converts it to `internal_graph`. If `pre=True` is used, the key is still `graph`.